topo_processor/data/data_transformers/data_transformer_imagery_historic.py

```diff
@@ -28,13 +29,15 @@ async def transform_data(self, item: Item) -> None:
         output_path = os.path.join(item.collection.temp_dir, f"{ulid.ulid()}.tiff")
         await create_cog(item.source_path, output_path, compression_method="lzw").run()
 
-        checksum = await multihash_as_hex(output_path)
-        asset = {
-            "path": output_path,
-            "key": "image",
-            "href": os.path.join(survey, f"{item.id}.tiff"),
-            "properties": {"file:checksum": checksum},
-            "stac_extensions": ["file"],
-            "content_type": stac.MediaType.COG,
-        }
-        item.add_asset(asset)
+        item.add_asset(
+            "cog",
+            Asset(
+                key="image",
+                path=output_path,
+                properties={"file:checksum": None},
```

Comment:
do you need to add the properties if there are none?

Reply:
no, properties is optional in stac asset. But I assumed we eventually will hash every asset so it will never be None?

Reply:
no, I have removed it now